lib/dor/text_extraction/speech_to_text_cocina_updater.rb

```diff
@@ -6,6 +6,25 @@ module TextExtraction
     class SpeechToTextCocinaUpdater < CocinaUpdater
       private
 
+      # we will skip .json speech to text files, these are only parsed for language to add to cocina
+      def skip_file?(file)
+        file.basename.to_s.start_with?('.') || file.basename.to_s.end_with?('.json')
```

Comment:
Unrelated: should we consider skipping everything but the .vtt and .txt files? Thinking about https://github.com/sul-dlss/speech-to-text/issues/41

Oh nevermind, I see that is done by controlling what output files are pulled back?

Reply:
Yes, that is a good idea since we could be even more careful here --- instead of adding everything to cocina except certain things, we could add only certain things to cocina.  And similar for OCR.

And yes, this is also guarded against in what we pull back from S3 but no harm in being extra cautious.

Reply:
Made this inclusionary instead of exclusionary (so only specific files are included).  Changed method name to match.